test/api.js

```diff
@@ -2886,7 +2886,7 @@ describe('/v1', function() {
           assert.equal(res.result.client_id, clientId);
           assert.equal(res.result.scope[0], 'profile');
           assert.equal(res.result.email, undefined);
-          assert.equal(res.result.profile_changed_at, PROFILE_CHANGED_AT_LATER_TIME, 'profile changed at is correct');
+          assert.equal(res.result.profile_changed_at, undefined, 'profile changed at is not set');
```

Comment:
I wonder if it's worth a comment on this test to explain why it's not set, in case we don't end up getting to this work for a little while.